packages/gated-content/.gitignore

```diff
@@ -0,0 +1,26 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
curious, was this generated by some tool given has a lot of files that we can't have?